### PR TITLE
test: Add missing pip dependencies

### PR DESCRIPTION
**Pull request by lluiscampos**
They went missing in c25129566c4d00bc8ee20f87526969a33af937b0

### CODE REVIEW

**Comment by lluiscampos** [comment]
Build with only acceptance tests @ [915576194](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/915576194)

**Review by kacf — APPROVED** [review]

File: tests/acceptance/requirements_py3.txt [hunk 1]
```diff
@@ -8,8 +8,12 @@ certifi==2023.5.7
     # via requests
 charset-normalizer==2.1.1
     # via requests
+exceptiongroup==1.1.1
+    # via pytest
```
Comment (lluiscampos) [inline tests/acceptance/requirements_py3.txt:12]: This is wrong, actually. It should have been added. I'll update my local pytest and "compile" again this file

**Comment by lluiscampos** [comment]
New build @ [Pipeline 915588409](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/915588409)

**Comment by lluiscampos** [comment]
There was another missing dependency: `ubi-reader`.

New build @ [Pipeline 915676271](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/915676271)

**Comment by lluiscampos** [comment]
The _final_ build @ [Pipeline 915856117](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/915856117)

**Pull request merged**